gadget/gadget.go

```diff
@@ -1562,7 +1607,22 @@ func validateBareContent(vc *VolumeContent) error {
 	return nil
 }
 
-func validateFilesystemContent(vc *VolumeContent) error {
+func validateEMMCContent(name string, vc *VolumeContent) error {
+	// We only allow offset and size for the rpmb block
```

Comment:
Do we support rpmb? I do not think you can simply write a block of data in a random offset there, but maybe I'm missing something here.